src/actions/fileAction.js

```diff
@@ -0,0 +1,112 @@
+import apiClient from '../helpers/apiClient'
+import { urlFile, urlFolderFiles, urlUploadFile } from '../urls'
+import { getFolder } from './folderActions'
+import {
+  GET_ALL_FILES,
+  GET_FOLDER_FILES,
+  FILE_API_ERROR,
+  UPLOAD_FILE,
+  GET_FILE_DETAILS,
+  GET_FOLDER_FILES_PENDING,
+  GET_ALL_FILES_PENDING,
```

Comment:
as GET_ALL_FILES_PENDING will never be used so delete it and instead rename GET_FOLDER_FILES TO GET_ALL_FILES to be more intutive